kmerexpr/transcriptome_reader.py

```diff
@@ -34,17 +34,23 @@ def shred(seq, K):
 
 
 def transcriptome_to_x(
-    K, fasta_file, x_file, L =None, max_nz=500 * 1000 * 1000, float_t=np.float32, int_t=np.int32
+    K,
+    fasta_file,
+    x_file,
+    L=None,
+    max_nz=500 * 1000 * 1000,
+    float_t=np.float32,
+    int_t=np.int32,
 ):
     print("K =", K)
     print("fasta file =", fasta_file)
     print("target x file =", x_file)
     print("float type =", float_t)
     print("int type =", int_t)
-    M = 4**K
+    M = 4 ** K
```

Comment:
[Comment]
I generally prefer space around operators, but this looks strange to see it around the power operator.

Reply:
I think that was a change `black` made, I didn't do it manually